packages/ui-template-svelte/src/routes/index.svelte

```diff
@@ -8,7 +8,7 @@
   import Checklist from "../components/Checklist.svelte";
   import Rate from "../components/Rate.svelte";
   import { vc } from "../store";
-  import { create } from "@nlx-voice-compass/core";
+  import { create } from "@nlx-voice-compass/core/src";
```

Comment:
This shouldn't be necessary so long as we are packaging things right (in fact it might indicate that something is off with the lerna setup?).

Reply:
yeah, something is totally of, but I spent a lot of time on it and couldn't make it `bootstrap` better, so I stuck to this way, because it's working

Reply:
I'll have a look!